fungible-token/src/tests.rs

```diff
@@ -0,0 +1,322 @@
+use codec::Encode;
+use fungible_token_messages::*;
+use gstd::String;
+use gtest::{Program, System};
+const USERS: &'static [u64] = &[3, 4, 5];
+
+fn init_with_mint<'a>(sys: &'a System) {
```

Comment:
Is lifetime really needed here?

```suggestion
fn init_with_mint(sys: &System) {
```